src/Components/Canvas/Canvas.tsx

```diff
@@ -1,120 +1,83 @@
 import "./Canvas.css";
 import 'p5';
 import '../../Types/Figures';
-import { AnimatedFigure, CircleFigure, SquareFigure, TriangleFigure } from '../../Types/ProcessingFigures';
-import { SelectedShape, SelectedAnimation } from '../../Types/Figures';
+import { SelectedShape, SelectedAnimation, SketchData } from '../../Types/Figures';
 import P5Wrapper from 'react-p5-wrapper';
 import 'react-p5-wrapper';
+import { useState } from "react";
+import React from "react";
+import { Animation } from '../../Types/Animations/Animation';
+
 
 function sketch (p) {
+    let sketchData: SketchData = {
+        onPressed: false,
+        figs: [],
+        points: [],
```

Comment:
What's the difference between figs and points?

Reply:
I'm not sure, Marco had added this field to the sketch closure, and I wanted to keep just incase he still needed it. However, it should probably be renamed or at least given documentation.

Reply:
Im going to merge these changes though, so that anyone making animations has access to the updated layout of things.